grakn-test/test-integration/src/test/java/ai/grakn/test/rule/DistributionContext.java

```diff
@@ -60,11 +63,11 @@
 
     public static final Logger LOG = LoggerFactory.getLogger(DistributionContext.class);
 
-    private static final FilenameFilter jarFiles = (dir, name) -> name.toLowerCase().endsWith(".jar");
-    private static final Path ZIP = Paths.get("grakn-dist-" + GraknVersion.VERSION + ".zip");
-    private static final Path CURRENT_DIRECTORY = Paths.get(GraknSystemProperty.PROJECT_RELATIVE_DIR.value());
-    private static final Path TARGET_DIRECTORY = CURRENT_DIRECTORY.resolve(Paths.get("grakn-dist", "target"));
-    private static final Path DIST_DIRECTORY = TARGET_DIRECTORY.resolve("grakn-dist-" + GraknVersion.VERSION);
+    private static final String ZIP_FILENAME = "grakn-dist-" + GraknVersion.VERSION + ".zip";
+    private static final Path GRAKN_BASE_DIRECTORY = Paths.get(GraknSystemProperty.PROJECT_RELATIVE_DIR.value());
+    private static final Path TARGET_DIRECTORY = Paths.get(GRAKN_BASE_DIRECTORY.toString(), "grakn-dist", "target");
```

Comment:
We can avoid breaking the `Path` abstraction with `toString` by using `Path#resolve`:
```
private static final Path TARGET_DIRECTORY = GRAKN_BASE_DIRECTORY.resolve(Paths.get("grakn-dist", "target"));
```
There are a few other places where we can do the same thing.

...but it's also a bit less pretty, so it's up to you if you want to do it or not!

Reply:
Yeah that is true. I switched the style since in my personal opinion it is a bit less readable :D

Reply:
so could `ZIP_FILENAME` use the `Paths.get()` instead of concatenating strings?

Reply:
@lolski yeah, I was half-half on which looks better too. I don't think there's much difference, so I'm good with this.